### DISCUSSION TITLE
Sharing Images/Links to App

### DISCUSSION THREAD

**Discussion by emiliegrimshaw** [quasarframework/quasar#10099] [CLI - Capacitor / Cordova modes]
Hi All!

Ideally, I'd like my users to be able to find an image or more specifically a piece of clothing from an online store and have the ability to share it with my application. I'm trying to source some info on the best way to do this. . This is largely out of my current skill base, so I'm not even sure where to start.

Similar to a social sharing link I need this to grab the central image and the header or title details, to be shared into my user's feed/page. Is this something someone has done before?

**Comment by ontwikkelfabriek** [comment]
@emiliegrimshaw 

> Is this something someone has done before?

Yes it has been done before with Quasar.

But the possibilities depend on what `mode` you are using in your Quasar app? Cordova/Capacitor/PWA/SPA/Electron/BEX/SSR ?

**Reply by emiliegrimshaw** [reply]
Currently building in SPA, I'm just doing some initial research into Capacitor as well :)
